test/python/unittest/API/APITest.py

```diff
@@ -447,6 +447,161 @@ def testConstantRecords(self):
+        )
+        self.assertEqual(
+            ms["LONG"][SCALAR].shape,
+            [0 for _ in range(5)]
+        )
+        self.assertEqual(
+            ms["LONGLONG"][SCALAR].shape,
+            [0 for _ in range(6)]
+        )
+        self.assertEqual(
+            ms["USHORT"][SCALAR].shape,
+            [0 for _ in range(7)]
+        )
+        self.assertEqual(
+            ms["UINT"][SCALAR].shape,
+            [0 for _ in range(8)]
+        )
+        self.assertEqual(
+            ms["ULONG"][SCALAR].shape,
+            [0 for _ in range(9)]
+        )
+        self.assertEqual(
+            ms["ULONGLONG"][SCALAR].shape,
+            [0 for _ in range(10)]
+        )
+        self.assertEqual(
+            ms["FLOAT"][SCALAR].shape,
+            [0 for _ in range(11)]
+        )
+        self.assertEqual(
+            ms["DOUBLE"][SCALAR].shape,
+            [0 for _ in range(12)]
+        )
+        self.assertEqual(
+            ms["LONG_DOUBLE"][SCALAR].shape,
+            [0 for _ in range(13)]
+        )
+
+        if found_numpy:
+            self.assertEqual(
+                ms["int16"][SCALAR].shape,
+                [0 for _ in range(14)]
+            )
+            self.assertEqual(
+                ms["int32"][SCALAR].shape,
+                [0 for _ in range(15)]
+            )
+            self.assertEqual(
+                ms["int64"][SCALAR].shape,
+                [0 for _ in range(16)]
+            )
+            self.assertEqual(
+                ms["uint16"][SCALAR].shape,
+                [0 for _ in range(17)]
+            )
+            self.assertEqual(
+                ms["uint32"][SCALAR].shape,
+                [0 for _ in range(18)]
+            )
+            self.assertEqual(
+                ms["uint64"][SCALAR].shape,
+                [0 for _ in range(19)]
+            )
+            self.assertEqual(
+                ms["single"][SCALAR].shape,
+                [0 for _ in range(20)]
+            )
+            self.assertEqual(
+                ms["np_double"][SCALAR].shape,
+                [0 for _ in range(21)]
+            )
+
+        # test datatypes for fixed-sized types only
+        if found_numpy:
+            self.assertTrue(ms["int16"][SCALAR].dtype == np.dtype("int16"))
+            self.assertTrue(ms["int32"][SCALAR].dtype == np.dtype("int32"))
+            self.assertTrue(ms["int64"][SCALAR].dtype == np.dtype("int64"))
+            self.assertTrue(ms["uint16"][SCALAR].dtype == np.dtype("uint16"))
+            self.assertTrue(ms["uint32"][SCALAR].dtype == np.dtype("uint32"))
+            self.assertTrue(ms["uint64"][SCALAR].dtype == np.dtype("uint64"))
+            self.assertTrue(ms["single"][SCALAR].dtype == np.dtype("single"))
+            self.assertTrue(
+                ms["np_double"][SCALAR].dtype == np.dtype("double"))
+
+    def testEmptyRecords(self):
+        backend_filesupport = {
+            'json': 'json',
+            'hdf5': 'h5',
+            'adios1': 'bp'
```

Comment:
Missing?
```suggestion
            'adios1': 'bp',
            'adios2': 'bp'
```

Reply:
```
In [2]: openpmd_api.variants
Out[2]: {'adios1': False, 'adios2': True, 'hdf5': False, 'json': True, 'mpi': True}
```
The first suggestion is the right one, yep I forgot ADIOS2.